flight/extract_gps.py

```diff
@@ -0,0 +1,94 @@
+"""Contains the extract_gps() function for extracting data out of
+the provided waypoint data JSON file for the SUAS competition."""
+from collections import namedtuple
+from typing import Any, List, NamedTuple
+import json
+from typing_extensions import TypedDict
+
+GPSData = TypedDict(
+    "GPSData",
+    {
+        "waypoints": List[NamedTuple],
+        "boundary_points": List[NamedTuple],
+        "altitude_limits": List[int],
+    },
+)
+
+
+def extract_gps(path: str) -> GPSData:
+    """Returns the waypoints, boundary points, and altitude limits from a waypoint data file.
+
+    Args:
+        path (str): File path to the waypoint data JSON file.
```

Comment:
Also need to reformat this

```suggestion
        path : str
                File path to the waypoint data JSON file.
```